app/main.ipynb

```diff
@@ -100,18 +102,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import numpy as np\n",
+    "\n",
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
+    "\n",
+    "subsequence = array[[1, 2, 3, 4, 5]]\n",
+    "\n",
+    "result = subsequence.sum()\n",
+    "print(result)"
```

Comment:
The subsequence sum is incorrect. Double-check the indices used for slicing to ensure they align with the task requirements. The expected sum is 27.